Libplanet/Net/ITransport.cs

```diff
@@ -1,49 +1,162 @@
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.Contracts;
 using System.Threading;
 using System.Threading.Tasks;
 using Libplanet.Net.Messages;
 
 namespace Libplanet.Net
 {
-    internal interface ITransport : IDisposable
+    /// <summary>
+    /// An interface to handle p2p, includes trading <see cref="Message"/>
+    /// and managing <see cref="Peer"/>s. Implements <see cref="IDisposable"/>.
```

Comment:
I'm not sure we should comment that it implements `IDisposable` through XML documentation.